package.json

```diff
@@ -211,6 +211,48 @@
       {
         "command": "terraform.plan",
         "title": "Terraform: plan"
+      },
+      {
+        "command": "terraform.modules.refreshList",
+        "title": "Refresh",
+        "icon": "$(refresh)"
+      },
+      {
+        "command": "terraform.modules.documentation",
+        "title": "Open Documentation",
+        "icon": "$(book)"
+      }
```

Comment:
Good catch. They didn't show up for me in the palette because I was looking for commands prefixed by `terraform`.  I didn't put `terraform` in the title for these since they're context specific, so therefore it didn't show up.

Reply:
I think it is worth keeping `terraform.command.name` as a convention for all commands. Since there isn't 'one place' to register all commands (package.json and then any TS file), you need a way to search for commands. Keeping a convention helps searching files (whether through grep or through VS Code) and helps visually identifying things.

Introducing a 'it gets a `terraform` prefix if it's in the command palette otherwise not` doesn't seem like it would help searching for commands that are not command palette commands.

Too make language even more fun with overloaded words, they are all commands anyway regardless of how they are visualized to the user

Reply:
I see, that's reasonable, thank you for the extra context 👌🏻  

I initially thought about `terraform` here as Terraform Core/CLI, but I guess we _can_ treat it as the name of the extension for filtering and other purposes.

> I didn't put terraform in the title for these since they're context specific, so therefore it didn't show up.

Does that mean these commands remain available in the command palette, just not as easy to find? Just trying to understand what happens if someone actually finds them and executes them without any context. Is there any way we can explicitly hide them from the palette as they are only relevant within the view container?

On a separate note I still think that we want to _avoid_ `terraform` in command IDs on the language server side unless that command actually executes Terraform. We already prefix commands there with `terraform-ls.` and the IDs there don't necessarily have the same UX problems as extensions have, would you agree?

Reply:
> Does that mean these commands remain available in the command palette, just not as easy to find?

Good question, sorry for the confusion. In short, yes. Each section in the `menu` block can determine when and how a command is displayed. If there is no `when` clause to prevent it, they are automatically displayed in the palette even if not listed in the `commandPalette` block. Think of the `commands` block as declaring the method signature, but the `menu` block determines the visibility (public|private) or the presentation of the method (title or alt text).

I hid these by creating entries for them in the `menu/commandPalette` section, and set their `when` clause to `false`. 

> On a separate note I still think that we want to avoid terraform in command IDs

I see your point. Let me review how we setup our commands on both sides and get back to you. This might be a separate ticket to reconcile conventions.

Reply:
Is `command: '${handler.commandPrefix}.terraform-ls.module.calls',` more inline with what you were expecting for the command sent to the LS?

Reply:
Yes, that is pretty much what I was expecting. 👍🏻